src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -166,7 +168,7 @@ public String toString() {
         Closed      // indicates that the connection has been closed.
     }
 
-    private final static float TIMEOUTSTEP = 0.08F;    // fraction of timeout to use for fast failover connections
+    private final static float TIMEOUTSTEP = 0.125F;    // fraction of timeout to use for fast failover connections
```

Comment:
We want to retain the timeslicing behavior for MSF and not change the timeslice values there. 
For TNIR you will have to define another step.